controller/app.post.controller.js

```diff
@@ -0,0 +1,19 @@
+const {
+    insertComment
+} = require("../model/app.post.model.js")
+
+
+exports.postComment = (req, res, next) => {
+
+    const {article_id} = req.params
+    const commentToAdd = req.body
+
+    insertComment(article_id, commentToAdd).then((postedComment) => {
+        res.status(201).send({
+            message: "comment posted",
```

Comment:
sending back the comment should be enough for the backend.
Showing comment posted is more something you'll do on the front-end to inform the user.